config/config-package.py

```diff
@@ -328,6 +328,12 @@ def gitignore(self):
             ignore=git_ignore,
         )
 
+    def readthedocs(self):
+        self.copy_with_meta(
+            'readthedocs.yaml.j2', self.path / '.readthedocs.yaml',
```

Comment:
A `git add` call is missing for this new file.

Reply:
added